sensing/pointcloud_preprocessor/include/pointcloud_preprocessor/filter.hpp

```diff
@@ -185,7 +197,14 @@ class Filter : public rclcpp::Node
   virtual void filter(
     const PointCloud2ConstPtr & input, const IndicesPtr & indices, PointCloud2 & output) = 0;
 
+  // Temporary Implementation: Remove this interface when all the filter nodes conform to new API.
+  // It's not pure virtual function so that a child class does not have to implement this function.
```

Comment:
```suggestion
  // TODO(sykwer): Temporary Implementation: Remove this interface when all the filter nodes conform to new API.
  // It's not pure virtual function so that a child class does not have to implement this function.
```